jquery.contextmenu.js

```diff
@@ -1,115 +1,113 @@
 /**
- * jQuery plugin for Pretty looking right click context menu.
+ * Forked by Joshua from  https://github.com/joewalnes/jquery-simple-context-menu
  *
- * Requires popup.js and popup.css to be included in your page. And jQuery, obviously.
- *
- * Usage:
- *
- *   $('.something').contextPopup({
- *     title: 'Some title',
- *     items: [
- *       {label:'My Item', icon:'/some/icon1.png', action:function() { alert('hi'); }},
- *       {label:'Item #2', icon:'/some/icon2.png', action:function() { alert('yo'); }},
- *       null, // divider
- *       {label:'Blahhhh', icon:'/some/icon3.png', action:function() { alert('bye'); }, isEnabled: function() { return false; }},
- *     ]
- *   });
- *
- * Icon needs to be 16x16. I recommend the Fugue icon set from: http://p.yusukekamiyamane.com/ 
- *
- * - Joe Walnes, 2011 http://joewalnes.com/
- *   https://github.com/joewalnes/jquery-simple-context-menu
+ * Now at
  *
  * MIT License: https://github.com/joewalnes/jquery-simple-context-menu/blob/master/LICENSE.txt
  */
 jQuery.fn.contextPopup = function(menuData) {
-	// Define default settings
-	var settings = {
-		contextMenuClass: 'contextMenuPlugin',
-		gutterLineClass: 'gutterLine',
-		headerClass: 'header',
-		seperatorClass: 'divider',
-		title: '',
-		items: []
-	};
-	
-	// merge them
-	$.extend(settings, menuData);
+    // Define default settings
+    //items should match dataTokenLookups!!
+    var settings = {
+        contextMenuClass: 'contextMenuPlugin',
+        fauxLinkTag: 'span',
+        fauxLinkAdditionalClass: 'contextMenuLink',
+        gutterLineClass: 'gutterLine',
+        headerClass: 'header',
+        seperatorClass: 'divider',
+        title: '',
+        items: [],
+        dataTokenLookups: []
+    };
+
+    // merge them
+    $.extend(settings, menuData);
```

Comment:
In the future, please don't make functional changes and whitespace/formatting changes in the commit. It makes it much harder to review. Better to send these as separate pull requests.